src/raft/recv_install_snapshot.h

```diff
@@ -3,8 +3,47 @@
 #ifndef RECV_INSTALL_SNAPSHOT_H_
 #define RECV_INSTALL_SNAPSHOT_H_
 
+#include <sqlite3.h>
+
 #include "../raft.h"
 
+// Forward declaration.
+struct snapshot_leader_state;
+
+struct snapshot_leader_io {
```

Comment:
Probably we will end up putting these in the fsm vtable or the io vtable or some mix of those.

Reply:
Agree, it is convenient for now to have them here for the tests.